hasher-matcher-actioner/hmalib/common/configs/actioner.py

```diff
@@ -48,43 +56,81 @@ class WebhookActionPerformer(ActionPerformer):
     headers: str
 
     def perform_action(self, match_message: MatchMessage) -> None:
-        self.call(data=json.dumps(match_message.to_aws()))
-
-    def call(self, data: str) -> Response:
+        parsed_url, parsed_headers = self.url, self.headers
+        for (
+            replacement_str,
+            replacement_func,
+        ) in WEBHOOK_ACTION_PERFORMER_REPLACEMENTS.items():
+            parsed_url = parsed_url.replace(
+                replacement_str, replacement_func(match_message)
+            )
+            parsed_headers = parsed_headers.replace(
+                replacement_str, replacement_func(match_message)
+            )
+        self.call(parsed_url, parsed_headers, data=json.dumps(match_message.to_aws()))
```

Comment:
if content_id/key and content_hash are already a part of data why add them to the header?

Reply:
I can imagine a use case where users might want this. It doesn't seem to add any more complexity so I figure why not include it